status-im/status-react
# Swipe-to-reply gesture

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)


[Fixes #12361 ] Swipe-to-reply gesture

### Summary
When a user swipes on a textual chat message, the reply input will be launched and populated with the message being replied to. The swiped message will also show a spring animation.

[comment]: # (Summarise the problem and how the pull request solves it)

### Review notes
Although I have added this functionality to an image chat message I have not really tested while replying to an image as I have not seen an image on a public chat to try and reply to. I will test this and confirm that it works as expected

### Testing notes
- Click on any public chat
- Swipe left on a text message
- See that the reply input has been populated and the swiped message has a spring animation for visual feedback upon swiping

#### Platforms




- Android
- iOS
- macOS
- Linux
- Windows

#### Areas that maybe impacted


##### Functional

- 1-1 chats
- public chats
- group chats
- wallet / transactions
- dapps / app browsing
- account recovery
- new account
- user profile updates
- networks
- mailservers
- fleet
- bootnodes

##### Non-functional

- battery performance
- CPU performance / speed of the app
- network consumption

### Steps to test


- Open Status
- ...
- Step 3, etc.



status: ready
